object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,13 @@
+/* exported omit */
+
+function omit(source, keys) {
+  var copy = Object.assign({}, source);
+  for (var k = 0; k < keys.length; k++) {
+    for (var key in copy) {
```

Comment:
You should be able to do this without a nested `for...in` loop here. You should check if there is a value in `copy` under the current property in the loop, and if so, delete it.